tests/create-action.test.js

```diff
@@ -0,0 +1,104 @@
+// Remove double quotes around snapshots.
+// This improves snapshot readability for generated file contents.
+expect.addSnapshotSerializer({
+  test: value => typeof value === 'string',
+  print: value => value
+})
+
+jest.mock('fs')
+jest.mock('enquirer')
+
+let mockFs
+
+const runCLI = (...args) => {
+  const createAction = require('../bin/create-action')
+  return createAction(args)
+}
+
+beforeEach(() => {
+  jest.resetModules()
+  mockFs = require('fs')
+
+  process.exit = jest.fn()
```

Comment:
The tests are failing on Node 11 because I'm assigning to a read-only property.

```
 FAIL  tests/create-action.test.js
  ● Test suite failed to run

    TypeError: Cannot assign to read only property 'Symbol(Symbol.toStringTag)' of object '#<process>'

      at _default (node_modules/jest-util/build/createProcessObject.js:85:34)
```

I'm not sure how to resolve yet. Did some googling - found https://github.com/facebook/jest/issues/2227, but the solutions in that thread didn't help me. 🤔

Reply:
Ah, see the fix in https://github.com/facebook/jest/pull/8050. Would require installing `jest@beta` for the time being.

Reply:
Yeah, definitely not related. Let's hang tight, this'll take me a little bit to fully review anyway 👍